crates/sui-replay/src/replay.rs

```diff
@@ -1684,6 +1684,48 @@ impl ChildObjectResolver for LocalExec {
             );
         res
     }
+
+    fn receive_object_at_version(
```

Comment:
Just curious, how is this tested. Do we have automated tests for replay?

Reply:
I don't believe this is tested in PR CI. However, the nightly backwards compatibility checks do use this code so it is tested fairly regularly (just not in PR CI).